OJS.Workers.ExecutionStrategies/Java/JavaSpringAndHibernateProjectExecutionStrategy.cs

```diff
@@ -501,5 +429,39 @@ private string ExtractEntryPointFromPomXml(string submissionFilePath)
             FileHelpers.DeleteFiles(pomXmlPath);
             return packageName.InnerText.Trim();
         }
+
+        private string GetMavenErrorsComment(string testOutput)
+        {
+            var sb = new StringBuilder();
+
+            foreach (var line in testOutput.Split(new[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries)
+                         .Where(x => x
+                             .StartsWith("[ERROR]") || x.StartsWith("[FAILURE]")))
```

Comment:
You have extracted `MavenErrorFailurePattern`, why not reuse it here. Also [FAILURE] could be added to the pattern and used like a regex.